src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -2295,64 +2295,6 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
             sGameEventMgr->StartEvent(eventId, true);
             break;
         }
-        case SMART_ACTION_START_CLOSEST_WAYPOINT:
```

Comment:
This reverts https://github.com/Riztazz/TrinityCore/commit/6066ec0949185cce92302ebf802149c916af11cf . Could you explain more in detail what you mean with "nothing users it atm" ? No SAI in TDB uses it ? (still, that poor guy who added it might use it)